backend/src/email/utils/callback/parsers/ses.ts

```diff
@@ -248,15 +263,21 @@ const parseRecord = async (record: SesRecord): Promise<void> => {
       type,
     })
     if (isTransactional) {
-      return EmailTransactionalService.handleStatusCallbacks(type, messageId, {
-        timestamp: new Date(record.Timestamp),
-        bounce: message.bounce,
-        complaint: message.complaint,
-        delivery: message.delivery,
-      })
+      return EmailTransactionalService.handleStatusCallbacks(
+        type,
+        messageId,
+        {
+          timestamp: new Date(record.Timestamp),
+          bounce: message.bounce,
+          complaint: message.complaint,
+          delivery: message.delivery,
+        },
+        parseRecordSpan
+      )
     }
     return parseNotificationAndEvent(type, message, metadata)
```

Comment:
Just confirming - you don't need to trace this

Reply:
I think I initially removed it because it wasn't working as I expected. I realised thats because it was executing asynchronously so it doesn't have a proper parent span. But I'll just add this in first and we can reason about it when we need to.